settings.json

```diff
@@ -1,3 +1 @@
-{
-  "dark_theme": false
-}
+{"dark_theme":false}
```

Comment:
Is this being reverted automatically? 🤔 Can we pretty-print application settings?